helperAPI.py

```diff
@@ -535,6 +535,7 @@ def getDriver(DOCKER=False):
         options.add_argument("--disable-blink-features=AutomationControlled")
         options.add_argument("--disable-extensions")
         options.add_argument("--disable-notifications")
+        options.add_argument("--log-level=3")
```

Comment:
Why are we setting log level to warn mode? I would prefer to keep this to what it is if we can.

Reply:
- Selenium prints out alot of logs that are unnecessary and not needed for what we are doing. These are not logs for the script but for the webdriver. I disagree that they are needed.